minorminer/utils/parallel_embeddings.py

```diff
@@ -0,0 +1,630 @@
+            match the first embedding :code:`embs[0].keys()`. node_order
+            can define any subset of the source graph nodes (embedding
+            keys).
+        as_ndarray: The list is cast to an ndarray, this requires that
+            the embedding is 1 to 1, that the embedding values (nodes of
+            a target graph) can be cast as numpy numeric types.
+
+    Raises:
+        ValueError: If `embs` is empty and `node_order` not provided.
+
+    Returns:
+        An embedding matrix; each row defines an embedding ordered
+            by node_order.
+    """
+    if node_order is None:
+        if len(embs) == 0:
+            if as_ndarray:
+                raise ValueError("shape of ndarray cannot be inferred")
+            else:
+                return []
+        else:
+            node_order = embs[0].keys()
+    if as_ndarray:
+        if len(embs) == 0:
+            return np.empty(shape=(0, len(node_order)))
+        else:
+            return np.asarray([[emb[v] for v in node_order] for emb in embs])
+    else:
+        return [[emb[v] for v in node_order] for emb in embs]
+
+
+def find_multiple_embeddings(
+    S: nx.Graph,
+    T: nx.Graph,
+    *,
+    max_num_emb: int = 1,
+    use_filter: bool = False,
+    embedder: callable = None,
+    embedder_kwargs: dict = None,
+    one_to_iterable: bool = False,
+    shuffle_all_graphs: bool = False,
+    seed: Union[int, np.random.RandomState, np.random.Generator] = None,
+) -> list:
+    """Finds multiple disjoint embeddings of a source graph onto a target graph
+
+    Uses a greedy strategy to deterministically find multiple disjoint
+    embeddings of a source graph onto a target graph; nodes used are removed
+    after each successful embedding.
+
+    Embedding multiple times on a large graph can take significant time.
+    It is recommended the user adjust embedder_kwargs appropriately such
+    as timeout, and also consider bounding the number of embeddings
+    returned (with max_num_emb).
+
+    Args:
+        S: The source graph to embed.
+        T: The target graph in which to embed.
+        max_num_emb: Maximum number of embeddings to find.
+            Defaults to 1, set to float('Inf') to find the maximum possible
+            number.
+        use_filter: Specifies whether to check feasibility
+            of embedding arguments independently of the embedder method.
+        embedder: Specifies the embedding search method,
+            a callable taking `S`, `T`, as the first two parameters. Defaults to
+            `minorminer.subgraph.find_subgraph`.
+        embedder_kwargs: Additional arguments for the embedder
+            beyond `S` and `T`.
+        one_to_iterable: Determines how embedding mappings are
+            interpreted. Set to `True` to allow multiple target nodes to map to
+            a single source node. Defaults to `False` for one-to-one embeddings.
+        shuffle_all_graphs: If True, the tile-masked target graph and source graph are
```

Comment:
```suggestion
        shuffle_all_graphs: If ``True``, the tile-masked target graph and source graph are
```
and also a few missing double back ticks in this docstring.